andela/ah-the-phoenix-frontend
# #163388093 Setup SASS pre-processing

**What does this PR do?**
Install and setup SASS pre-processing

**Description of Task to be completed?**
- install SASS
- change css extensions to .scss
  
**How should this be manually tested?**

-Go to the branch ch-setup-SASS-processing-163388093
- Run npm start and check in your browser if existing styling files work on the page

**Any background context you want to provide?**

**What are the relevant pivotal tracker stories?**
[#163388093](https://www.pivotaltracker.com/story/show/163388093)
